oneanddone/tasks/models.py

```diff
@@ -92,9 +92,27 @@ def has_feedback(self):
             return False
 
     @property
+    def needs_verification(self):
+        return self.task.must_be_verified and self.state == self.STARTED
+
+    @property
     def next_task(self):
         return self.task.next_task
 
+    @property
+    def verification_status(self):
+        if (not len(self.communication_set.all()) or
```

Comment:
`not self.communication_set.exists()` is cheaper. `len(..all())` will force all results from db to be loaded and then counted in python.